identity-and-access-management/audit-trail/how-to/configure-audit-trail-permissions.mdx

```diff
@@ -0,0 +1,32 @@
+---
+meta:
+  title: How to configure the Audit Trail permission set
+  description: Learn how to configure IAM permissions for accessing Scaleway's Audit Trail, enabling event tracking across Organizations and Projects.
+content:
+  h1: How to configure the Audit Trail permission set
+  paragraph: Learn how to configure IAM permissions for accessing Scaleway's Audit Trail, enabling event tracking across Organizations and Projects.
+tags: audit-trail events tracking iam permissions
+dates:
+  validation: 2024-11-26
+  posted: 2024-11-26
+categories:
+  - identity-and-access-management
+---
+
+This page teaches you how to configure the necessary permission set to access [Audit Trail](/identity-and-access-management/audit-trail/concepts/#audit-trail).
+
+## How to configure the Audit Trail permission set
+
+To start using Audit Trail you need to configure the `AuditTrailReadOnly` permission set in [IAM](/identity-and-access-management/iam/concepts/#iam). The [scope](/identity-and-access-management/iam/concepts/#scope) of this permission set is at [Organization](/identity-and-access-management/iam/concepts/#organization) level.
```

Comment:
There is another permission set that will give the permission: "OrganizationManager"

Reply:
Which is included in the group "Administrator" that is created by default when Organizations get created